helpers/cache/Cache.js

```diff
@@ -0,0 +1,27 @@
+class Cache {
+	constructor(callback, { updateInvervalSecounds } = {}) {
+		this.lastUpdated = 0;
+		this.updateInvervalSecounds = (updateInvervalSecounds * 1000) || 60000;
+		this.data = null;
+		if (!(typeof callback === 'function')) {
+			throw new Error('First parameter in "Cache" constructor must be a function.');
+		}
+		this.callback = callback;
+	}
+
+	async get(req, res, next) {
+		if (Date.now() > (this.lastUpdated + this.updateInvervalSecounds)) {
+			// please first update time and response old data until request return result
```

Comment:
do you need please here? :)

Reply:
yes :P

Reply:
please update time first. further requests are waiting on the promise.